app/app.go

```diff
@@ -398,10 +398,7 @@ func (app *App) RemoveUnits(n uint) error {
 	if n == 0 {
 		ReleaseApplicationLock(app.Name)
 		return stderr.New("Cannot remove zero units.")
-	} else if l := uint(len(app.Units())); l == n {
-		ReleaseApplicationLock(app.Name)
-		return stderr.New("Cannot remove all units from an app.")
-	} else if n > l {
+	} else if l := uint(len(app.Units())); n > l {
```

Comment:
The change looks good, but I'd like to take this opportunity to rename this variable to "length". Using ``l`` as a variable name is confuse :-(

And I'm sorry, because I think I wrote this code in the first place. Do you mind just renaming this variable? After that, we can merge the pull request.